<launch>
  <node name="order_topic_publisher" pkg="ros_msg_test"  type="order.cpp"/>
  <node name="topic_publisher" pkg="ros_msg_test" type="topic_publisher.cpp" />
  <node name="topic_subscriber" pkg="ros_msg_test" type="topic_subscriber.cpp"/>
</launch>
